bin/rialto/runtime/src/lib.rs

```diff
@@ -436,6 +436,7 @@ parameter_types! {
 	pub const HeadersToKeep: u32 = 7 * bp_rialto::DAYS as u32;
 }
 
+pub type MillauGrandpaInstance = ();
```

Comment:
For more clarity, shouldn't this rather be:
```suggestion
pub type MillauGrandpaInstance = pallet_bridge_grandpa::DefaultInstance;
```

Reply:
the `pallet_bridge_grandpa` not have `DefaultInstance`, I'm use `cargo docs` generated docs, I'm not found it. 

![image](https://user-images.githubusercontent.com/37804932/116543827-ee7ab980-a920-11eb-99e3-49a38ea9baea.png)

![image](https://user-images.githubusercontent.com/37804932/116543855-f8042180-a920-11eb-9729-cdb68634c9e2.png)

Reply:
It should be `()` instead of `DefaultInstance`. `DefaultInstance` was removed in FRAME `v2`.